nomad/local/vault-agent-conf.hcl

```diff
@@ -0,0 +1,18 @@
+log_level = "Info"
+log_requests_level = "info"
+ui = true
+
+api_addr = "http://127.0.0.1:8200"
+
+#service_registration "consul" {
```

Comment:
ugh, so there's a circular dependency here. We need vault to configure consul, but that means we can't use the service_registration stanza here since that expects consul to be ready immediately 💩 
I'm looking for cli commands to do this now